.gitignore

```diff
@@ -1,4 +1,134 @@
+# This is generated from running bin/fink_client_test.sh ,
+# not sure if this should be tracked or not
+schemas/distribution_schema.avsc
```

Comment:
indeed, this one should not be tracked (downloaded from the Fink servers or created on-the-fly).

Reply:
Resolved in [`e85b4a0`](https://github.com/astrolabsoftware/fink-client/pull/22/commits/e85b4a0150c7881dd656c9e436ac6efd2c046925)